1/Dockerfile

```diff
@@ -48,6 +48,36 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod 775 /etc/passwd && \
```

Comment:
i'm having second thoughts about this since it means anyone who can run a job on jenkins can update the passwd file which in turn means they can have free run of the jenkins container, including changing the jenkins config to give themselves more access.  (for example I think they could add/update an existing user with a different password and then su to that user)

it would be better if we can "close the door behind us" after we update the file on startup but i don't think that's going to be possible.

going to have to think about this one some more.